WalletLibrary/WalletLibrary/Networking/Entities/OpenID4VCI/CredentialConfiguration.swift

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+/**
+ * The constraints for issuance and display information of a credential,
+ */
+struct CredentialConfiguration: Decodable
+{
+    /// The Verified ID issued.
+    let format: String?
+    
+    /// The notification id to pass to notification endpoint.
+    let scope: String?
+    
+    /// The crypto binding methods supported (ex. did:jwk)
+    let cryptographic_binding_methods_supported: [String]?
```

Comment:
Does all these have to be String or is there a standard list that we can use as enum so that we can avoid errors due to different ways of representing them? This applies to all the list of strings in data models below.

Reply:
This data model is really for just serialization. My goal for this new component is completely splitting up serialization and validation of params, signatures, etc. Which will allow for more extensibility in the future. A Verifier can thereotically but anything in this property and we will need to validate that we support the binding, crypto suite, etc in another layer.